docs/components/checkbox-field/demo/base-demo.md

```diff
@@ -0,0 +1,23 @@
+```hbs template
+<Form::CheckboxField
+  @label='Label'
+  @value={{this.value}}
+  @onChange={{this.updateValue}}
+/>
+```
+
+```js component
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class extends Component {
+  @tracked value = false;
+  @tracked errorMessage;
```

Comment:
Suggestion: remove this errorMessage...or use it, since it's not being used.

Reply:
Thanks for catching this! Applied your feedback